src/net35/Radical/Observers/BrokerObserver.cs

```diff
@@ -77,9 +80,9 @@ public MessageBrokerMonitor( IMessageBroker broker )
         /// </summary>
         /// <typeparam name="TMessage">The type of the message.</typeparam>
         /// <returns>This monitor instance.</returns>
-        public MessageBrokerMonitor WaitingFor<TMessage>() where TMessage : class
+        public MessageBrokerMonitor WaitingFor<TMessage>( InvocationModel invocationModel = InvocationModel.Default ) where TMessage : class
```

Comment:
I fix it, then we will squash it just before merge it with the new github feature:

![image](https://cloud.githubusercontent.com/assets/1633595/14645465/0c0a8110-0656-11e6-89f1-be0822ddd48e.png)